Dockerfile-database

```diff
@@ -1,5 +1,7 @@
 FROM centos/postgresql-10-centos7
 
+ENV APP_VERSION=latest
```

Comment:
should this variable be updated with every release? it needs to be updated from new-release script then

Reply:
ok